### PR TITLE
Add affinity and tolerations for loki workloads

### PR DESCRIPTION
**Pull request by ath88**
#### What does this PR do?
Adds affinity and tolerations for loki workloads

#### What are the relevant tickets?
86

### CODE REVIEW

File: infrastructure/environments/dplplat01/configuration/loki/loki-values.template.yaml [hunk 1]
```diff
@@ -75,3 +156,26 @@ monitoring:
         operator: Exists
       - key: noderole.dplplatform
         operator: Exists
+  selfMonitoring:
+    grafanaAgent:
+      tolerations:
+        - key: CriticalAddonsOnly
+          operator: Exists
+      # grafana operator agent does not support affinity in this version of the chart - should be fixed later
```
Comment (ITViking) [inline infrastructure/environments/dplplat01/configuration/loki/loki-values.template.yaml:164]: why is this commented out?

Comment (ath88) [reply infrastructure/environments/dplplat01/configuration/loki/loki-values.template.yaml:164]: uh - see comment I wrote.
> grafana operator agent does not support affinity in this version of the chart - should be fixed later

Comment (ath88) [reply infrastructure/environments/dplplat01/configuration/loki/loki-values.template.yaml:164]: theres no support for setting affinity on the pod

**Review by ITViking — APPROVED** [review]
good to go

**Pull request merged**